flink-table/flink-table-planner-blink/pom.xml

```diff
@@ -150,24 +150,29 @@ under the License.
 			<groupId>org.apache.calcite</groupId>
 			<artifactId>calcite-core</artifactId>
 			<!-- When updating the Calcite version, make sure to update the dependency exclusions -->
-			<version>1.19.0</version>
+			<version>1.20.0</version>
 			<exclusions>
 				<!--
-
-				Dependencies that are not needed for how we use Calcite right now.
-
-				"mvn dependency:tree" as of Calcite 1.19:
-
-				[INFO] +- org.apache.calcite:calcite-core:jar:1.19.0:compile
-				[INFO] |  +- org.apache.calcite.avatica:avatica-core:jar:1.13.0:compile
-				[INFO] |  +- org.apache.calcite:calcite-linq4j:jar:1.19.0:compile
-				[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-core:jar:2.9.6:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.6:compile
-				[INFO] |  +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.6:compile
+				"mvn dependency:tree" as of Calcite 1.20:
```

Comment:
Do we have to depend on all of these? It increased a lot just for upgrading calcite version. Could you remove useless dependency?

Reply:
i just list all calcite-core dependencies here. i found that some dependencies are not in the original list, and i revert the useless dependencies.

Reply:
This list should reflect all compile dependency, I'm quite doubt for dependency like "net.hydromatic:aggdesigner-algorithm"

Reply:
the full dependency tree:
```
mvn dependency:tree | grep ":compile"
[INFO] +- commons-codec:commons-codec:jar:1.12:compile
[INFO] +- org.apache.calcite.avatica:avatica-core:jar:1.15.0:compile
[INFO] |  +- org.apache.calcite.avatica:avatica-metrics:jar:1.15.0:compile
[INFO] |  +- com.google.protobuf:protobuf-java:jar:3.6.1:compile
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.6:compile
[INFO] |  \- org.apache.httpcomponents:httpcore:jar:4.4.10:compile
[INFO] +- org.apache.calcite:calcite-linq4j:jar:1.21.0-SNAPSHOT:compile
[INFO] +- org.apache.commons:commons-dbcp2:jar:2.6.0:compile
[INFO] |  \- commons-logging:commons-logging:jar:1.2:compile
[INFO] +- org.apache.commons:commons-lang3:jar:3.8:compile
[INFO] +- com.esri.geometry:esri-geometry-api:jar:2.2.0:compile
[INFO] +- com.fasterxml.jackson.core:jackson-core:jar:2.9.8:compile
[INFO] +- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.8:compile
[INFO] +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.8:compile
[INFO] +- com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:jar:2.9.8:compile
[INFO] |  \- org.yaml:snakeyaml:jar:1.23:compile
[INFO] +- com.google.code.findbugs:jsr305:jar:3.0.1:compile (optional) 
[INFO] +- com.google.guava:guava:jar:19.0:compile
[INFO] +- com.yahoo.datasketches:sketches-core:jar:0.9.0:compile
[INFO] |  \- com.yahoo.datasketches:memory:jar:0.9.0:compile
[INFO] +- com.jayway.jsonpath:json-path:jar:2.4.0:compile
[INFO] +- net.hydromatic:aggdesigner-algorithm:jar:6.0:compile
[INFO] |  \- commons-lang:commons-lang:jar:2.4:compile
[INFO] +- org.codehaus.janino:janino:jar:3.0.11:compile
[INFO] +- org.codehaus.janino:commons-compiler:jar:3.0.11:compile
[INFO] +- org.slf4j:slf4j-api:jar:1.7.25:compile
```
i revert the dependency back to before the upgrade, and just update the version.